docs/reference/commandline/service_create.md

```diff
@@ -384,9 +402,23 @@ volumes in a service:
      </p>
     </td>
   </tr>
-</table>
+  <tr>
+    <td><b>bind-nonrecursive</b></td>
+    <td>
+      By default, submounts are recursively bind-mounted as well. However, this behavior can be confusing when a
+      bind mount is configured with <tt>readonly</tt> option, because submounts are not mounted as read-only.
+      Set <tt>bind-nonrecursive</tt> to disable recursive bind-mount.<br />
+
+      A value is optional:
 
-#### Bind Propagation
```

Comment:
Looks like you accidentally removed this heading (should probably be added back below the table)